client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/packages-step/package-info.js

```diff
@@ -81,7 +82,10 @@ const PackageInfo = props => {
 		}
 
 		return (
-			<Button className="packages-step__add-item-btn" compact onClick={ onAddItem }>
+			<Button
+				className={ classNames( 'button', 'packages-step__add-item-btn' ) }
```

Comment:
is `compact` just secondary? How come this one doesn't need compact?

Reply:
Good catch. Added `is-compact`.